oracle/prices.js

```diff
@@ -3,6 +3,9 @@ const coinUtils = require('./utils.js').utils;
 const axios = require('axios');
 
 var getCoinGeckoPrice = async (marketID) => {
+  if (marketID === "busd:usd" || marketID === "busd:usd:30" ) {
```

Comment:
nit: use a constant variable; or a map as it will allow us to easily add more stable coin markets in the future